Mathlib/CategoryTheory/Limits/Preserves/Basic.lean

```diff
@@ -241,6 +241,21 @@ lemma preservesLimitOfPreservesLimitCone {F : C ⥤ D} {t : Cone K} (h : IsLimit
     (hF : IsLimit (F.mapCone t)) : PreservesLimit K F :=
 preservesLimit_of_preserves_limit_cone h hF
 
+lemma preservesLimit_of_equiv
```

Comment:
I would think this should be stated for a functor `G : J' ⥤ J` which satisfies `G.IsEquivalence`, but this is probably unnecessary because if you import `CategoryTheory.Limits.Final`, the conclusion of the lemma is already an instance.

Reply:
Thank you, I was indeed just missing an import :)